frappe/translations/zh.csv

```diff
@@ -1,4230 +0,0 @@
-apps/frappe/frappe/utils/change_log.py,New {} releases for the following apps are available,可以使用以下应用程序的新{}版本
```

Comment:
@MorezMartin Why are you deleting translation files of other languages?

Reply:
By mistake

Reply:
I pulled develop branch, created a new branch from develop and merge my translate_dekstop branch in this new branch but same problem. Can you help me on doing it the good way ?
Thanks in advance,
Sincerely yours,
Martin